mne/io/brainvision/brainvision.py

```diff
@@ -420,7 +306,7 @@ def _get_eeg_info(vhdr_fname, eog, misc, response_trig_shift):
     if settings.find('[Comment]') != -1:
         params, settings = settings.split('[Comment]')
     else:
-        params, settings = settings, str()
+        params, settings = settings, ''
```

Comment:
I don't think we use `str()` in many other places in the codebase, so I slightly prefer `''` for consistency.